exporters/committime/app.py

```diff
@@ -117,9 +117,13 @@ class GitCommittimeConfig:
     def __attrs_post_init__(self):
         if not (self.username and self.token):
```

Comment:
actually, this implementation is wrong, no?

 I think the `and` should be a `or` here, because the `elif` is never called like this